liquibase-standard/src/main/java/liquibase/executor/Executor.java

```diff
@@ -126,9 +126,9 @@ public interface Executor extends Plugin {
      */
     int queryForInt(SqlStatement sql, List<SqlVisitor> sqlVisitors) throws DatabaseException;
 
-    List queryForList(SqlStatement sql, Class elementType) throws DatabaseException;
+    <T> List<T> queryForList(SqlStatement sql, Class<T> elementType) throws DatabaseException;
```

Comment:
I think this one as it's not specifying any particular type should be fine. cc: @filipelautert